concert/app/src/layer/python/enchanted_brain/attributes.py

```diff
@@ -1,5 +1,6 @@
 ATTR_RECORD_ID = "recordId"
 ATTR_CREATED_AT = "createdAt"
+ATTR_STAGE_ID = "stageId"
```

Comment:
Please call this `ATTR_EVENT_STAGE_ID` to be consistent with record-type-specific attribute names.

Reply:
Done.